frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.js

```diff
@@ -834,25 +782,56 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
     this.setState({ volumeTracingToDownsample: null });
   };
 
+  showAddVolumeLayerModal = () => {
+    this.setState({ isAddVolumeLayerModalVisible: true });
+  };
+
+  hideAddVolumeLayerModal = () => {
+    this.setState({ isAddVolumeLayerModalVisible: false });
+  };
+
   render() {
     const { layers } = this.props.datasetConfiguration;
-    const layerSettings = Object.entries(layers).map(entry => {
-      const [layerName, layer] = entry;
-      const isColorLayer = getIsColorLayer(this.props.dataset, layerName);
-      // $FlowIssue[incompatible-call] Object.entries returns mixed for Flow
-      return this.getLayerSettings(layerName, layer, isColorLayer);
-    });
+    const layerSettings = _.sortBy(
```

Comment:
Sure thing! It just sorts the layers so that the color layers come first. At some point, I experimented with having a visual "section" for the color layers and one section for segmentation layers. However, it looked weird when only having one layer in one section, so I only kept the sorting (and added a comment now).